.gitignore

```diff
@@ -1,4 +1,6 @@
 .project
 .settings
 .idea
+/vendor
+/composer.lock
```

Comment:
This PR looks like that it will not hold the cached libraries when executing the `composer install` command.
It looks good to me.